### PR TITLE
Removing selenium dependencies from pom.xlm

### PR DESCRIPTION
**Pull request by Aayush-Srikanth**
Removing selenium dependencies from pom.xlm

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,18 +15,6 @@
     </properties>
```
Comment (SerineBetraoui) [inline pom.xml:16]: :)

File: pom.xml [hunk 2]
```diff
@@ -15,18 +15,6 @@
...
 
     <dependencies>
-        <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
-        <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
```
Comment (Leosaw) [inline pom.xml:21]: We didn't need this so that fine

**Comment by SerineBetraoui** [comment]
high level comment

File: pom.xml (hunk 2 above)
Comment (Leosaw) [reply pom.xml:21]: Awesome good to go

File: pom.xml [hunk 3]
```diff
@@ -15,18 +15,6 @@
     </properties>
 
     <dependencies>
-        <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
```
Comment (SuzyLady) [inline pom.xml:18]: looks fine to me

**Review by SuzyLady — APPROVED** [review]

File: pom.xml (hunk 3 above)
Comment (Aayush-Srikanth) [reply pom.xml:18]: amazing!

**Review by jacktegart — APPROVED** [review]
Ready for API Testing :))

**Review by SerineBetraoui — APPROVED** [review]
**I am happy with this**

**Review by Leosaw — APPROVED** [review]
awesome

**Pull request merged**